bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        use_stdout=True,
+        use_stderr=True)
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License', 'Copyright Notice'}
+
+    _output_regex = re.compile(
+        r'(?P<message>\S+.*)')
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=60):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.
+        """
+
+        return '--lines', str(licensecheck_lines), filename
+
+    def process_output(self, output, filename, file):
```

Comment:
you still override `process_output`, don't you? :D

I think this is possible instead:
```python
@linter(
    executable='licensecheck',
    use_stdout=True,
    # use_stderr=True,  <--- you don't need stderr btw !!!
    output_format='regex',
    output_regex=r'.*: \*No copyright\* UNKNOWN',
    result_message='No license found.' # <-- I think 'No license found' sounds somehow better than 'License not found', can't fully explain you why, so it's up to you +1
```
then you don't need `_output_regex` being defined manually in the class + you don't have to override `process_output` :+1: